src/static/css/iframe_editor.css

```diff
@@ -38,6 +38,17 @@ body.grayedout { background-color: #eee !important }
   color: black;
 }
 
+@media screen and (-webkit-min-device-pixel-ratio:0){
```

Comment:
I'm no frontend person.

If the bug is chrome specific, should'n this `-webkit-min-device-pixel-ratio` only apply to Chrome?

Then again, from my distant past I think to remember that it is better to do capability detection instead of hand-patching for specific browsers.

@seballot, can you have a look? This is related to issue #3487.

Reply:
This wrapping thing seems not easy ! From ace2_inner.js

```javascript
    // Chrome can't handle the truth..  If CSS rule white-space:pre-wrap
    // is true then any paste event will insert two lines..
    // Sadly this will mean you get a walking Caret in Chrome when clicking on a URL
    // So this has to be set to pre-wrap ;(
    // We need to file a bug w/ the Chromium team.
    if(browser.chrome){
      $("#innerdocbody").addClass("noprewrap");
    }
```

And from ifram_editor.css
```css
body {
  white-space: nowrap;
  word-wrap: normal;
}
body.doesWrap {
  display: block !important;
  /* white-space: pre-wrap; */
  /*
    Must be pre-wrap to keep trailing spaces. Otherwise you get a zombie caret,
    walking around your screen (see #1766).
    WARNING: Enabling this causes Paste as plain text in Chrome to remove line breaks
    this is probably undesirable
    WARNING: This causes copy & paste events to lose bold etc. attributes
    NOTE: The walking-zombie caret issue seems to have been fixed in FF upstream
    so let's try diabling pre-wrap and see how we get on now.
    For more details see: https://github.com/ether/etherpad-lite/issues/2574
  */
  word-wrap: break-word; /* fix for issue #1648 - firefox not wrapping long lines (without spaces) correctly */
}
.noprewrap{
  white-space: normal;
}
body.doesWrap:not(.noprewrap) > div {
  /* Related to #1766 */
  white-space: pre-wrap;
}
```

What I think is that all those problems have been fixed in recent browsers, and we could simplify it and use `white-space: pre-wrap;` everywhere. But it would need a bit more investigation and testing